draft-ietf-teep-protocol.md

```diff
@@ -450,6 +450,7 @@ token
   value MUST correspond to the value received with the QueryRequest message
   if one was present, and MUST be absent if no token was present in the
   QueryRequest.
+  Refer the QueryRequest Message section for other requirements of token handling.
```

Comment:
I think it would be better to move the text out of the QueryRequest Message section and into a "Token Handling" section, maybe another subsection under "Creating and Validating TEEP Messages"